### DISCUSSION TITLE
SDK 1.7.2?

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#737] [Q&A] [answered]
Hello Labs,

I am glad to see the new SDK 1.9.1 release (what maybe is not in use now), however I would like to know when do you want to release the update of the old versions what in use? 
We are waiting it very much. I think I am not alone with it.

We use SDK 1.7.1. 
This version is full of bugs (eg.schematron rules) and PDF display errors, not to mention bad translations. We were waiting for the official translations because you have promised it to June. We make a lot of workaround and fixed the SDK but  there are lot of bug what we cannot fix, only your developer team.

We reported a lot of bugs since early spring but didn't get any fix for them under SDK 1.7.
Now the our official PDF files will be full of bad translations and there are lots of data, what cannot see in the generated PDF files. 

We have to go live with SDK version 1.7.1 in the coming days. The deadline is deadline we cannot change it.
We cannot update our SDK bigger version, you know well these process.

With the greatest respect and understanding I would like to ask, when can we expect the SDK 1.7.2 (with bug fixes and good hungarian translations)?

With the greatest respect to you and your work
Anna

**Comment by YvesJo** [comment, marked as answer]
For this non-technical question please get in touch with the helpdesk
